py/torch_tensorrt/dynamo/conversion/impl/shuffle.py

```diff
@@ -120,3 +130,60 @@ def pixel_unshuffle(
         permuted_tensor,
         shape[:-3] + (out_channels, out_height, out_width),
     )
+
+
+def resize_(
```

Comment:
I think the function name `resize`, instead of `resize_`, is good enough, for consistency.

Reply:
Thank you for your review. I have addressed your comment.